<launch>

  <arg name="cam_dev" default = "/dev/video2" />
  <arg name="cam_info_path" default = "ost.yaml"/>
  <node pkg="colab_ros_camera_node" type="camera_info_publisher.py" name="camera_info" args="$(find colab_ros_camera_node)/config/$(arg cam_info_path)"/>
  <node pkg="colab_ros_camera_node" type="camera_image_publisher.py" name="camera_image" args="$(arg cam_dev)"/>

</launch>
